gui-win10/SvgConverter.cs

```diff
@@ -439,7 +530,7 @@ public XAttribute[] ParseStyleValue(string eleName, string styleValue)
                 string name = nameAndValue[0];
                 string value = nameAndValue[1];
                 
-                if (ALLOWED_ELEMENTS_NAME.Contains(eleName) == true & ALLOWED_ATTRIBUTES[eleName].Contains(name) == false)

```

Comment:
Aside from the allowed elements change, previous code usage of single ampersand meant that if ALLOWED_ELEMENTS_NAME.Contains was false, the latter half would get resolved and throw an exception.